server/lib/config/nginx.conf

```diff
@@ -71,7 +71,7 @@ http {
 
     server {
         listen       8080;
-        listen       [::]:8080;
+#        listen       [::]:8080;
```

Comment:
Do we want to be retaining this commented-out line?

Reply:
Ouch. I didn't mean to commit that, but I missed it. Sadly, I need this on my IPv6-deprived laptop, unless I change the boot configuration; but we don't want to propagate it. (Although we're not using IPv6 addresses in the lab, and I don't think this config file goes to the production deployment anyway, so I doubt it really matters much.)

Reply:
And ... missed it *again*, after delaying fixing it so I could *test*, and then forgetting. 😦